apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/RODViewPoolCandidatePage.tsx

```diff
@@ -0,0 +1,523 @@
+        description: "Title for the career timeline snapshot section",
+      }),
+    },
+    personal: {
+      id: "personal",
+      title: intl.formatMessage({
+        defaultMessage: "Personal and contact information",
+        id: "0lUoqK",
+        description:
+          "Title for the personal and contact information snapshot section",
+      }),
+    },
+    work: {
+      id: "work",
+      title: intl.formatMessage({
+        defaultMessage: "Work preferences",
+        id: "s7F24X",
+        description: "Title for the work preferences snapshot section",
+      }),
+    },
+    dei: {
+      id: "dei",
+      title: intl.formatMessage({
+        defaultMessage: "Diversity, equity, and inclusion",
+        id: "zLeH2i",
+        description:
+          "Title for the diversity, equity, and inclusion snapshot section",
+      }),
+    },
+    government: {
+      id: "government",
+      title: intl.formatMessage({
+        defaultMessage: "Government employee information",
+        id: "nEVNHp",
+        description:
+          "Title for the government employee information snapshot section",
+      }),
+    },
+    language: {
+      id: "language",
+      title: intl.formatMessage({
+        defaultMessage: "Language profile",
+        id: "KsS1Py",
+        description: "Title for the language profile snapshot section",
+      }),
+    },
+    signature: {
+      id: "signature",
+      title: intl.formatMessage({
+        defaultMessage: "Signature",
+        id: "1ZZgbi",
+        description: "Title for the signature snapshot section",
+      }),
+    },
+  };
+
+  const subTitle = (
+    <div
+      data-h2-display="l-tablet(flex)"
+      data-h2-align-items="base(center)"
+      data-h2-justify-content="base(space-between)"
+      data-h2-margin="base(x1, 0)"
+    >
+      {snapshotUserPropertyExists && (
+        <>
+          <Heading
+            level="h3"
+            data-h2-margin="base(0)"
+            data-h2-font-weight="base(800)"
+          >
+            {sections.snapshot.title}
+          </Heading>
+          <ToggleGroup.Root
+            type="single"
+            color="primary.dark"
+            value={preferRichView ? "text" : "code"}
+            onValueChange={(value) => {
+              if (value) setPreferRichView(value === "text");
+            }}
+          >
+            <ToggleGroup.Item value="text">
+              {intl.formatMessage({
+                defaultMessage: "Text",
+                id: "Ude1JQ",
+                description: "Title for the application's profile snapshot.",
+              })}
+            </ToggleGroup.Item>
+            <ToggleGroup.Item value="code">
+              {intl.formatMessage({
+                defaultMessage: "Code",
+                id: "m0JFE/",
+                description: "Title for the application's profile snapshot.",
+              })}
+            </ToggleGroup.Item>
+          </ToggleGroup.Root>
+        </>
+      )}
+    </div>
+  );
```

Comment:
I think we are removing this featrue, @Jerryescandon can you confirm that we no longer support toggling between a rich view and code view?

If we do plan to support it, this currently only appears when  `preferRichView=false` so it actually never shows up.

Reply:
@Jerryescandon is the text/code toggle getting removed on the new ROD view pool candidate (application) page?

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/33ac3683-b172-4d70-ad7a-0ffb1328b865)